### PR TITLE
Added Pillpack

### PR DESCRIPTION
**Pull request by Possibly6757**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -13956,7 +13956,17 @@
...
     },
-
+    
+    {
+        "name": "Pillpack",
+        "url": "https://help.pillpack.com/hc/en-us/requests/new",
+        "difficulty": "hard",
+        "notes": "Will need to contact customer support and confirm your name and information before they'll delete.",
+        "domains": [
+            "https://www.pillpack.com/"
```
Comment (Kefaku) [inline _data/sites.json:13966]: There are more domains:

```suggestion
            "pillpack.com",
            "app.pillpack.com",
            "folks.pillpack.com",
            "my.pillpack.com"
```

**Review by Kefaku — CHANGES_REQUESTED** [review]

File: _data/sites.json [hunk 2]
```diff
@@ -13956,7 +13956,20 @@
             "picpay.me"
         ]
     },
-
+    
+    {
+        "name": "Pillpack",
```
Comment (tupaschoal) [inline _data/sites.json:13961]:
```suggestion

    {
        "name": "Pillpack",
```

File: _data/sites.json [hunk 3]
```diff
@@ -13956,7 +13956,20 @@
...
+        "notes": "Will need to contact customer support and confirm your name and information before they'll delete.",
+        "domains": [
+            "pillpack.com",
+            "app.pillpack.com",
+            "folks.pillpack.com",
+            "my.pillpack.com"
+        ]
+    },
+    
     {
```
Comment (tupaschoal) [inline _data/sites.json:13973]:
```suggestion
    },

    {
```

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**